solutions/src/main/java/ru/sberbank/school/task13/cacheproxy/Cache.java

```diff
@@ -0,0 +1,21 @@
+package ru.sberbank.school.task13.cacheproxy;
+
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Cache {
+    enum cacheType {MEMORY, FILE}
```

Comment:
Type name &#39;cacheType&#39; must match pattern &#39;^[A-Z][a-zA-Z0-9]*$&#39;.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1716871882>*